images/base/files/usr/local/bin/entrypoint

```diff
@@ -193,7 +193,7 @@ enable_network_magic(){
   sed -e "s/${docker_embedded_dns_ip}/${docker_host_ip}/g" /etc/resolv.conf.original >/etc/resolv.conf
 
   # fixup IPs in manifests ...
-  curr_ipv4="$( (getent ahostsv4 $(hostname) | head -n1 | cut -d' ' -f1) || true)"
+  curr_ipv4="$( (getent ahostsv4 "$(hostname)" | head -n1 | cut -d' ' -f1) || true)"
```

Comment:
is not a problem the nested double quotes?

Reply:
it works on my pc, it should be fine then :)

Reply:
in shell double quotes nest the same as parens: balanced quotes close. this is because double quotes are a context in which things expand.

single quotes do not.

Reply:
not having these double quotes before means we word-split on hostname's output.
now hostnames should not have spaces, but it's still functionally incorrect bash to not have the quotes here.